recipe/meta.yaml

```diff
@@ -51,7 +51,7 @@ about:
   home: http://www.mlpack.org
   license: BSD-3-Clause
   license_file: LICENSE.txt
-  summary: mlpack: a fast, header-only machine learning library
+  summary: mlpack, a fast, header-only machine learning library
```

Comment:
Might also work to quote the statement

```suggestion
  summary: "mlpack: a fast, header-only machine learning library"
```

Reply:
Ah, nice, much better.  Let's find out if it works!

Reply:
Seems to work :)

Asked the bot to re-render below to propagate this change through

Reply:
Sweet.  Thanks!